streams/src/main/java/org/apache/kafka/streams/processor/internals/RecordCollectorImpl.java

```diff
@@ -136,20 +136,13 @@ public void flush() {
         checkForException();
```

Comment:
I guess this can be fixed independently, and it would be cleaner not to have it in this PR, but just to do it with KAFKA-5006. Thanks!